### ISSUE TITLE
Exact command to obtain base waterbirds model

### ISSUE THREAD

**Issue by alexlioralexli** [PolinaKirichenko/deep_feature_reweighting#1]
I'm trying to reproduce the 74.9% worst case and 98.1% average test accuracy on Waterbirds, but could not do so. I'm running the command below. 
```
python3 train_classifier.py --output_dir=<OUTPUT_DIR> --pretrained_model \
  --num_epochs=100 --weight_decay=1e-3 --batch_size=32 --init_lr=1e-3 \
  --eval_freq=1 --data_dir=<WATERBIRDS_DIR> --test_wb_dir=<WATERBIRDS_DIR> \
  --augment_data --seed=<SEED> --num_minority_groups_remove=0
```

**Comment by rahulv54** [comment]
Were you able to?

**Comment by PolinaKirichenko** [comment]
Hi! Which worst-group and average accuracies do you get instead? @alexlioralexli @rahulv54

**Comment by rahulv54** [comment]
I did it sometime back. Let me check again.

**Comment by Haoxiang-Wang** [comment]
I ran your example command of ERM (base model) in README for Waterbirds, and I can only get average accuracy of 90-93% instead of >=97% (reported in your paper). Could you try re-running the codebase & provide the exact commands for the reproduction of your results? Thank you! @PolinaKirichenko @izmailovpavel @andrewgordonwilson

**Comment by izmailovpavel** [comment]
Hey @Haoxiang-Wang, the average accuracy in the paper (and in general reported in the papers in this area) is the accuracy according to the training group distribution. I.e. you compute accuracy on each group and average these accuracies with weights corresponding to the proportions of groups in the training data. 

Our training script does not output average accuracy, it simply outputs the accuracy on the test set. In waterbirds, the test set is close to uniform across groups, so the test set accuracy is lower than the average accuracy.
